README.md

```diff
@@ -100,11 +100,79 @@ this can be used with `watch`.
 watch -c juju status --color
 ```
 
-### Alternate deployment methods
+## Alternate deployment methods
 
+### Deploying storage
 
+The Canonical Distribution of Kubernetes allows you to connect with durable
+storage devices such as [Ceph](http://ceph.com). When paired with the
+[Juju Storage](https://jujucharms.com/docs/2.0/charms-storage) feature you
+can add durable storage easily and across all clouds.
 
-#### Usage with your own binaries
+Deploy the ceph-mon and ceph-osd charms.
+
+```
+juju deploy cs:ceph-mon -n 3
+juju deploy cs:ceph-osd -n 3
+```
+
+Relate the charms:
+```
+juju add-relation ceph-mon ceph-osd
+```
+
+List the storage pools available to Juju for your cloud:
```

Comment:
suggest: Perhaps call out that this is output specific to AWS. If someone deploys to say azure, and attempts to enlist an EBS volume its going to give them a hard time.